maintenance-mode.php

```diff
@@ -44,3 +56,20 @@ function vip_maintenance_mode_load_plugin_textdomain() {
 	load_plugin_textdomain( 'maintenance-mode', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
 }
 add_action( 'init', 'vip_maintenance_mode_load_plugin_textdomain' );
+
+/**
+ * Displays a warning in the admin to inform users about the VIP_MAINTENANCE_MODE constant
+ *
+ * Only users who can activate plugins will see the warning.
+ *
+ * @since 0.1.1
+ */
+function vip_maintenance_mode_admin_notice__constant_not_set() {
```

Comment:
It'd be nicer if this hooked function was placed closer to the `add_action()` call on line 22 for better readability.